src/test/ui/privacy/multiple-private-fields.stderr

```diff
@@ -0,0 +1,34 @@
+error[E0451]: fields of struct `a::A` are private
+  --> $DIR/multiple-private-fields.rs:8:18
+   |
+LL |     pub struct A(usize, usize);
+   |     --------------------------- `a::A` defined here
+...
+LL |     let x = a::A(3, 4);
+   |             ---- ^  ^ private field
+   |             |    |
+   |             |    private field
```

Comment:
I think there's a way to only have a single message for multiple spans in one line